sdk/python/experimental/hello/requirements.txt

```diff
@@ -1 +1 @@
-dagger-io
+dagger-io[server]
```

Comment:
The `[server]` extra is needed to install Strawberry. It's configured here:

https://github.com/dagger/dagger/blob/0c1d7fc40f99c43a1e0b33107d8513417c928706/sdk/python/pyproject.toml#L55-L58